pkg/otlp/internal/serializerexporter/exporter.go

```diff
@@ -102,6 +103,10 @@ func translatorFromConfig(logger *zap.Logger, cfg *exporterConfig) (*translator.
 		options = append(options, translator.WithInstrumentationLibraryMetadataAsTags())
 	}
 
+	if cfg.Metrics.ExporterConfig.InstrumentationScopeMetadataAsTags {
```

Comment:
I think if the user sets both `InstrumentationScopeMetadataAsTags` and `InstrumentationLibraryMetadataAsTags` we should error out and tell the user to only use the instrumentation scope one

Reply:
It's also fine to keep the current behavior, but then we should update the release note: the two options work independently, don't they?

Reply:
Do you mean it should return error like this?

```go
	if cfg.Metrics.ExporterConfig.InstrumentationLibraryMetadataAsTags && cfg.Metrics.ExporterConfig.InstrumentationScopeMetadataAsTags {
		return nil, fmt.Errorf("cannot use both instrumentation_library_metadata_as_tags(deprecated) and instrumentation_scope_metadata_as_tags")
	}
```

Reply:
yeah that makes sense!